### ISSUE TITLE
Setting up App Keys

### ISSUE THREAD

**Issue by 0x00trust** [ravgeetdhillon/custom-controllers-strapi#1]
cd backend
npm install
npm run develop

`npm run develop

> custom-controllers-strapi@0.1.0 develop
> strapi develop

Building your admin UI with development configuration ...
Admin UI built successfully

╭──────────────────────────────────────────────────────────────────────────────╮│                                                                              ││             A new version of Strapi is available 4.1.7 → 4.1.11              ││   Check out the new releases at: https://github.com/strapi/strapi/releases   ││                                                                              │╰──────────────────────────────────────────────────────────────────────────────╯

[2022-05-21 13:41:54.829] debug: ⛔️ Server wasn't able to start properly.
[2022-05-21 13:41:54.831] error: Middleware "strapi::session": App keys are required. Please set app.keys in config/server.js (ex: keys: ['myKeyA', 'myKeyB'])
Error: Middleware "strapi::session": App keys are required. Please set app.keys in config/server.js (ex: keys: ['myKeyA', 'myKeyB'])
    at instantiateMiddleware (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/services/server/middleware.js:12:11)
    at resolveMiddlewares (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/services/server/middleware.js:56:18)
    at registerApplicationMiddlewares (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/services/server/register-middlewares.js:66:29)
    at async Object.initMiddlewares (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/services/server/index.js:99:7)
    at async Strapi.bootstrap (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/Strapi.js:403:5)
    at async Strapi.load (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/Strapi.js:415:5)
    at async Strapi.start (/home/leo/Desktop/custom-controllers-strapi/backend/node_modules/@strapi/strapi/lib/Strapi.js:163:9)`

**Comment by ravgeetdhillon** [comment]
@0x00trust If you look in this [file](https://github.com/ravgeetdhillon/custom-controllers-strapi/blob/master/backend/config/server.js#L5), you need to set `APP_KEYS` environment variable in your `.env` file.

So, in the `.env` file, just add the following line:

```
APP_KEYS=jP8pb1lYsAhnmURarewxhA==,34xnLMYHY5jiU7ONTstTqQ==
```

You can change these random strings according to your own choice.

You can also use a tool like [Random Key Generator](https://www.coolgenerator.com/random-key-generator) for creating keys.

**Comment by 0x00trust** [comment]
Thank you
Let me try again

**Comment by 0x00trust** [comment]
Hi,
I have got this template
https://github.com/0x00trust/bholdus-web
Can you help me install
Thank you
